test/performance/alphabet/alphabet_to_char_benchmark.cpp

```diff
@@ -19,15 +19,35 @@
 #include <seqan/modifier.h>
 #endif
 
+template <typename alphabet_t, bool is_seqan2>
+std::array<alphabet_t, 256> create_alphabet_array()
+{
+    std::array<alphabet_t, 256> alphabet_array{};
+
+    auto convert_to_alphabet = [] (auto const c, auto & a)
+    {
+        if constexpr (is_seqan2)
+        {
+             a = (char) c;
+        }
+        else
+        {
+            using char_t = seqan3::alphabet_char_t<alphabet_t>;
+            seqan3::assign_char_to(char_t(c), a);
+        }
```

Comment:
```suggestion
        using char_t = seqan3::alphabet_char_t<alphabet_t>;
        if constexpr (is_seqan2)
             a = char{c};
        else
            seqan3::assign_char_to(char_t{c}, a);
```

Reply:
@rrahn  This does not work, because `using char_t = seqan3::alphabet_char_t<alphabet_t>;` would then be called when used seqan2, which means `seqan3::alphabet_char_t` is called for `seqan::SimpleType<unsigned char, seqan::Dna_>` and seqan3 does not know what to do with seqan2...